src/core/render/index.js

```diff
@@ -401,7 +402,7 @@ export function initRender(vm) {
   }
 
   const id = config.el || '#app';
-  const navEl = dom.find('nav') || dom.create('nav');
+  const navEl = dom.find(config.nav_el || 'nav') || dom.create('nav');
```

Comment:
The doc update says

> It can be a CSS selector string or an actual HTMLElement

however if an actual HTMLElement is passed in here, it will throw a runtime error, for example like this:

```
Uncaught DOMException: Failed to execute 'querySelector' on 'Document': '[object HTMLDivElement]' is not a valid selector.
```

We should cover this case in the tests, or update the docs to say it requires a selector string.

Reply:
I removed the mention of HTMLElement, but this is what the description for `config.el` says which uses the same `dom.find()` method

Reply:
@Evidlo Oh yeah, that's totally wrong. Thanks for pointing that out!